nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_2L_140FB-1_TOT/kinematics.yaml

```diff
@@ -0,0 +1,13 @@
+bins:
+- zero:
+    min: null
+    mid: 0
+    max: null
```

Comment:
There's no need to add a fake kinematic. Please remove the zero.

Fwiw, there's also no need to sdd sqrts or min/max.